src/Utilities/Util.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Tooltip } from '@patternfly/react-core';
+import { NO_DATA_VALUE, NO_DATA_STATE } from '../constants';
+import { SystemState } from '../Components/RosTable/SystemState';
+import { ProgressScoreBar } from '../Components/RosTable/ProgressScoreBar';
+import { InsightsLink } from '@redhat-cloud-services/frontend-components/InsightsLink';
+
+export const systemName = (displayName, id, { inventory_id: inventoryId, isDeleted, state }) => {
```

Comment:
Hello @r14chandra,
since these are methods related to rending the column on UI I feel it will be good to put them in [RenderColumn.js file](https://github.com/RedHatInsights/ros-frontend/blob/main/src/Components/RosTable/RenderColumn.js#L16) and not in separate Util for consistency and readability. In past I had already added the other 4 methods related to rending columns in it. Let me know what you think! 🙌🏻

Reply:
Thanks for pointing that out!